micro_sam/sample_data.py

```diff
@@ -36,6 +39,23 @@ def fetch_image_series_example_data(save_directory: Union[str, os.PathLike]) ->
     return data_folder
 
 
+def sample_data_image_series():
+    """Provides 2d image series example data to napari."""
+    # Return list of tuples
+    # [(data1, add_image_kwargs1), (data2, add_image_kwargs2)]
+    # Check the documentation for more information about the
+    # add_image_kwargs
+    # https://napari.org/stable/api/napari.Viewer.html#napari.Viewer.add_image
```

Comment:
This `sample_data.py` file was created

We won't have to do anything similar for the widgets - just adding type hints should be enough there. See [this example](https://github.com/napari/cookiecutter-napari-plugin/blob/6e1d851ad6a0e2b3a584366c4fdef3f3dcd433da/%7B%7Bcookiecutter.plugin_name%7D%7D/src/%7B%7Bcookiecutter.module_name%7D%7D/_widget.py#L37-L46)